src/Compilers/CSharp/Test/Emit3/Semantics/UnsafeEvolutionTests.cs

```diff
@@ -15,6 +16,52 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Semantics;
 [CompilerTrait(CompilerFeature.Unsafe)]
 public sealed class UnsafeEvolutionTests : CompilingTestBase
 {
+    private void CompileAndVerify(
+        string lib,
+        string caller,
+        object[] unsafeSymbols,
```

Comment:
~~`string[]` for `unsafeSymbols` and `safeSymbols`?~~ Consider asserting the allowed types for these parameters. It's not obvious from reading this method
nit: consider including "expected" in those parameter names